sevntu-checks/src/test/resources/com/github/sevntu/checkstyle/checks/coding/FieldDeclarationsCountCheckNOK.java

```diff
@@ -0,0 +1,12 @@
+package com.github.sevntu.checkstyle.checks.coding;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class FieldDeclarationsCountCheckNOK {
+
+    // both FQN and import'ed type names work:
+    Logger logger1;
```

Comment:
Examples:
```
~/java/github/sevntu-checkstyle/sevntu.checkstyle/sevntu-checks/src/main/java [master|✔] 
$ grep -R "IMPORT" * | head -n 5 
com/github/sevntu/checkstyle/checks/coding/ForbidInstantiationCheck.java:        return new int[] {TokenTypes.IMPORT, TokenTypes.LITERAL_NEW };
com/github/sevntu/checkstyle/checks/coding/ForbidInstantiationCheck.java:            case TokenTypes.IMPORT:
com/github/sevntu/checkstyle/checks/coding/ForbidInstantiationCheck.java:     *        literal node ("IMPORT" or "LITERAL_NEW" node types).
com/github/sevntu/checkstyle/checks/coding/ForbidInstantiationCheck.java:     *         "IMPORT" node or instanstiated class Name&Path for given
com/github/sevntu/checkstyle/checks/coding/ForbidCertainImportsCheck.java:                TokenTypes.IMPORT, TokenTypes.LITERAL_NEW, };
```
https://github.com/sevntu-checkstyle/sevntu.checkstyle/blob/master/sevntu-checks/src/main/java/com/github/sevntu/checkstyle/checks/coding/ForbidInstantiationCheck.java

Yes, it could be postponed but in this case please mention this in javadoc as limitation that could be improved in future.